lib/grape/validations/types/dry_type_coercer.rb

```diff
@@ -0,0 +1,36 @@
+require 'dry-types'
+
+module DryTypes
+  include Dry.Types()
```

Comment:
Remove `()`, or maybe just use `Dry.Types....` everywhere. It's liberally used below as either.

Reply:
I would love to do that, but it is how dry-types [should be used](https://dry-rb.org/gems/dry-types/1.2/getting-started/). After calling `Dry.Types()` [registered types get added to a container](https://github.com/dry-rb/dry-types/blob/master/lib/dry/types.rb#L160-L212), it involves adding [constants](https://github.com/dry-rb/dry-types/blob/master/lib/dry/types/module.rb#L102-L117) and some other meta-programming.

Reply:
Ok, I think I am missing something. I am staring at dry-types types.rb code and I don't get it. How does it make `Dry.Types` different from `Dry.Types()`?

Reply:
[Dry.Types](https://github.com/dry-rb/dry-types/blob/master/lib/dry/types.rb#L210) is a method which returns an instance of a module to be included. :slightly_smiling_face:  Here is [that module](https://github.com/dry-rb/dry-types/blob/master/lib/dry/types/module.rb#L20).